maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -529,3 +507,14 @@ class IOSDriver(
         private const val SCREEN_SETTLE_TIMEOUT_MS: Long = 3000
     }
 }
+
+private fun Double.asPercentOf(total: Int): Int {
```

Comment:
what's the advantage of this over the regular private functions from before?

Reply:
Conciseness, mainly.

If I were to keep using `widthPercentToPoint(percent: Double)` I'd have to refactor it to `widthPercentToPoint(percent: Double, width: Double)` which becomes a bit long and repetitive.